packages/flutter/lib/src/material/icon_button.dart

```diff
@@ -168,7 +171,8 @@ class IconButton extends StatelessWidget {
     return new InkResponse(
       onTap: onPressed,
       child: result,
-      radius: math.max(size, Material.defaultSplashRadius),
+      radius: math.max(48.0, iconSize + math.min(padding.horizontal, padding.vertical))
+          * 0.55, // x 0.5 for diameter -> radius and + 10% overflow so it doesn't look too square.
```

Comment:
Ya, you're totally right. The contacts app has a 34.33 radius on a presumably 48 icon. Will keep the 35 and also kept the 40% overflow for bigger sizes.